refactron/core/pipeline.py

```
@@ -0,0 +1,197 @@
+        for file_path_str, items in files_to_fix.items():
+            file_path = Path(file_path_str)
+            if not file_path.exists():
+                for item in items:
+                    item.status = FixStatus.BLOCKED
+                    item.block_reason = "file not found"
+                    session.blocked_fixes.append(item)
+                continue
+
+            issues_for_file = self._queue_items_to_issues(items, file_path)
+            try:
+                fixed_code, diff = self.fix_engine.fix_file(
```

Ruff (0.15.9)

[warning] Unpacked variable `fixed_code` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)